portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -307,14 +307,42 @@ definition {
 	}
 
 	@description = "Verify that a Text field's Predefined Value can be translated"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test TextFieldPredefinedValueCanBeTranslated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133494 TextFieldPredefinedValueCanBeTranslated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		Form.gotoAdvancedTab();
+
+		FormViewBuilder.setFieldPredefinedDate(fieldValue = "Predefined value");
```

Comment:
Hi @fmarcio, there is a macro called `editPredefinedValue` in the `FormFields.macro`, could you please use that one instead? Because the FormViewBuilder macro it is not from the Forms, and using FormFields.macro would minimize future refactor.

Reply:
> Hi @fmarcio, there is a macro called `editPredefinedValue` in the `FormFields.macro`, could you please use that one instead? Because the FormViewBuilder macro it is not from the Forms, and using FormFields.macro would minimize future refactor.

@mlan0 fixed. I used `editPredefinedValueInput` because `editPredefinedValue` did not work on my test